### ISSUE TITLE
XP Support

### ISSUE THREAD

**Issue by NEO-SPECTRUMAN** [TASEmulators/fceux#499]
Looks like fceux-2.6.4 works on XP well...
but there is 6.0 (vista) sub system version in PE Header
and it doesn't run on XP without PE header patching

**Comment by Dwedit** [comment]
The difference between an XP build and a Non-XP build is how thread-local storage is accessed.  In versions of Windows prior to Vista, you need to use TlsGetValue to get a global pointer.  But on Vista or later, there is a segmentation register that already points to the thread-local storage.  If you run an EXE built for Non-XP on XP, you will get a crash every time it tries to access thread-local storage.  So hence why there are separate builds for XP and Non-XP.

FCEUX probably doesn't make use of thread-local storage, so it might be okay to patch the PE header.  But you'd also need to check all the DLL dependencies for use of thread-local storage.

I personally think it doesn't make much sense to make 32-bit builds that aren't XP builds.

**Comment by zeromus** [comment]
fceux is supposed to be built with the vc140_xp toolchain. As far as I know, this is supposed to set subsystem 5.01. This happens via Toolset.props in the toolset on the msbuild in program files directory in the MinimumRequiredVersion element. Indeed when I built it just now, I got a 5.01 in the subsystem.
Easiest way to confirm is to go here and go to header > Optional Header and find MajorOperatingSystemVersion etc.
https://speedtesting.herokuapp.com/peviewer/
Why arent the public releases using 5.01? I don't know...
I'll make sure future releases use it.
